vsphere/internal/virtualdevice/virtual_machine_disk_subresource.go

```diff
@@ -1008,7 +1008,7 @@ func DiskPostCloneOperation(d *schema.ResourceData, c *govmomi.Client, l object.
 			//
```

Comment:
Can we prepend `UPSTREAM 1161:` to the commit title @mtnbikenc

Reply:
Updated the commit title.